FreeRTOS-Plus/Demo/AWS/Device_Shadow_Windows_Simulator/Common/main.c

```diff
@@ -107,7 +107,7 @@ static UBaseType_t ulNextRand;
 int main( void )
 {
     /***
-     * See https://www.FreeRTOS.org/mqtt_lts/index.html for configuration and usage instructions.
+     * See https://www.FreeRTOS.org/coremqtt for configuration and usage instructions.
```

Comment:
Was this supposed to be suffixed with `*/index.html` ?

Reply:
As per prior reply (the web browser will try opening index.html automatically so there is no need to add it).